wgs_somatic-run-wrapper.py

```diff
@@ -110,6 +113,66 @@ def analysis_end(outputdir, tumorsample, normalsample):
     else:
         pass
 
+
+def get_insilico_info(all_insilico=True, panel_names=[]):
```

Comment:
why are these insilico functions in the wrapper script? they are not used in the wrapper?